src/App.tsx

```diff
@@ -1,9 +1,10 @@
 import React from "react";
-import { Admin, Resource, ListGuesser } from "react-admin";
+import { Admin, Resource, EditGuesser } from "react-admin";
 import fakeDataProvider from "ra-data-fakerest";
 import generateData from "data-generator-retail";
 import { ProductList } from "./product/ProductList";
 import { CommandList } from "./command/CommandList";
+import { CommandEdit } from "./command/CommandEdit";
```

Comment:
Something we do often is to add an `index.ts` file in each resource directory and export an object having the same shape as the `Resource` props:

```
export default {
    list: CommandList,
    edit: CommandEdit
}
```

And we use it like this:

```
import commands from './command';

<Resource name="commands" {...commands} />
```

Reply:
Not very important though